st2common/st2common/content/loader.py

```diff
@@ -15,22 +15,59 @@
 
 import os
 
-import six
 from yaml.parser import ParserError
+import six
 
 from st2common import log as logging
 from st2common.constants.meta import ALLOWED_EXTS
 from st2common.constants.meta import PARSER_FUNCS
 from st2common.constants.pack import MANIFEST_FILE_NAME
+from st2common.constants.runners import MANIFEST_FILE_NAME as RUNNER_MANIFEST_FILE_NAME
 
 __all__ = [
+    'ContentRunnerLoader',
     'ContentPackLoader',
     'MetaLoader'
 ]
 
 LOG = logging.getLogger(__name__)
 
 
+class ContentRunnerLoader(object):
```

Comment:
As discussed stevedore is -1 and at the very least not happening in this PR. (I realize this is already decided, but I'm leaving comments on all existing comments so we can get this ready to close out.)